packages/bridge/src/browser.ts

```diff
@@ -0,0 +1,5 @@
+import bridge from './index'
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+window.condoBridge = bridge
```

Comment:
```suggestion
window['condoBridge'] = bridge
```

Reply:
The safest way: https://lodash.com/docs/4.17.15#set

Reply:
It'a bit weird to increase bundledize with lodash for this purpose

Reply:
Another way is declaring namespace with custom window object. But the simplest way is not use dot syntax here

Reply:
<img width="757" alt="image" src="https://user-images.githubusercontent.com/50069872/211737853-119ca618-c139-4702-bbfc-0dff68c3f967.png">

Reply:
Found elegant way to extend window interface